theme/src/components/homeSlider.js

```diff
@@ -1,29 +1,30 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { NavLink } from 'react-router-dom';
-import ImageGallery from 'react-image-gallery';
-import { themeSettings } from '../lib/settings';
-
-const renderItem = item => (
-	<div className="image-gallery-image">
-		<NavLink to={item.path || ''}>
-			<img src={item.original} alt={item.title} />
-			<div
-				className="caption"
-				style={{ color: themeSettings.home_slider_color || '#fff' }}
-			>
-				<div className="caption-title">{item.title}</div>
-				<div className="caption-description">{item.description}</div>
-			</div>
-		</NavLink>
-	</div>
-);
-
-const HomeSlider = ({ images }) => {
-	if (images && images.length > 0) {
-		const items = images.map(item => ({
-			original: `/assets/images/${item.image}`,
-			title: item.title,
+import React from 'react';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+import ImageGallery from 'react-image-gallery';
+import { themeSettings } from '../lib/settings';
+import storeSettings from '../../../config/store';
+
+const renderItem = item => (
+	<div className="image-gallery-image">
+		<NavLink to={item.path || ''}>
+			<img src={item.original} alt={item.title} />
+			<div
+				className="caption"
+				style={{ color: themeSettings.home_slider_color || '#fff' }}
+			>
+				<div className="caption-title">{item.title}</div>
+				<div className="caption-description">{item.description}</div>
+			</div>
+		</NavLink>
+	</div>
+);
+
+const HomeSlider = ({ images, settings }) => {

```

Comment:
settings is not used, i think it can be removed

Reply:
Thanks, removed